packages/uni_ui/lib/card_timeline.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:uni_ui/generic_card.dart';
+
+class CardTimeline extends StatelessWidget {
+  const CardTimeline(
+      {required this.startTime,
+      required this.endTime,
+      required this.card,
+      this.isActive = false,
+      super.key});
+
+  final String startTime;
+  final String endTime;
+  final GenericCard card;
```

Comment:
Are you sure we want to receive a generic card?

Reply:
A widget instead?

Reply:
> Are you sure we want to receive a generic card?

A Widget instead?

Reply:
yes, better be a Widget